bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html

```diff
@@ -0,0 +1,35 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<footer class="platform-footer">
+  <nav class="platform-footer__navigation">
+    <ul class="platform-footer__navigation--list">
+      <li class="platform-footer__navigation--list-item"><a>Blog</a></li>
+      <li class="platform-footer__navigation--list-item"><a>About</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Explore</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Publishers</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Developers</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Terms of Service</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Privacy</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Support</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Jobs</a></li>
+    </ul>
+    <div class="platform-footer__navigation--social-media">
+    <div class="platform-footer__navigation--social-media__wrapper">
+      <span class="platform-footer__navigation--social-media--text">Connect:</span>
+      <ul class="platform-footer__navigation--social-media--list">
+        <li class="platform-footer__navigation--social-media--list-item">
+          <a href="https://twitter.com/pocket" class="platform-footer__navigation--social-media--list-item-link twitter" aria-label="Twitter" target="_blank" rel="external noopener noreferrer"><span>Twitter</span></a>
+        </li>
+        <li class="platform-footer__navigation--social-media--list-item">
+          <a href="https://facebook.com/pocket" class="platform-footer__navigation--social-media--list-item-link facebook" aria-label="Facebook" target="_blank" rel="external noopener noreferrer"><span>Facebook</span></a>
+        </li>
+      </ul>
+    </div>
+      <p class="platform-footer__navigation--social-media--copyright">© 2020 Read It Later, Inc</p>
+    <div>
+  <nav>
```

Comment:
This should be a closing tag
```suggestion
  </nav>
```